spec/tapioca/generators/base_generator_spec.rb

```diff
@@ -0,0 +1,170 @@
+          result = generator.rbi_header("bin/tapioca dsl", reason: "foo")
+          expected = <<~RUBY
+            # DO NOT EDIT MANUALLY
+            # This is an autogenerated file for foo.
+            # Please instead update this file by running `bin/tapioca dsl`.
+
+          RUBY
+          assert_equal(expected, result)
+        end
+
+        it "displays the correct information when strictness is set" do
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
+          result = generator.rbi_header("bin/tapioca dsl", strictness: "true")
+          expected = <<~RUBY
+            # DO NOT EDIT MANUALLY
+            # This is an autogenerated file for .
+            # Please instead update this file by running `bin/tapioca dsl`.
+
+            # typed: true
+
+          RUBY
+          assert_equal(expected, result)
+        end
+
+        it "displays the correct information when reason and strictness are set" do
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
+          result = generator.rbi_header("bin/tapioca dsl", reason: "foo", strictness: "true")
+          expected = <<~RUBY
+            # DO NOT EDIT MANUALLY
+            # This is an autogenerated file for foo.
+            # Please instead update this file by running `bin/tapioca dsl`.
+
+            # typed: true
+
+          RUBY
+          assert_equal(expected, result)
+        end
+      end
+
+      describe "#dsl_rbi_filename" do
+        it "provides the correct filename for single-word constant names" do
+          constant = "Constant"
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
+          result = generator.dsl_rbi_filename(constant)
+          expected = config.outpath / "constant.rbi"
+          assert_equal(expected, result)
+        end
+
+        it "provides the correct filename for multi-word constant names" do
+          constant = "ThisIsMyTestConstant"
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
+          result = generator.dsl_rbi_filename(constant)
+          expected = config.outpath / "this_is_my_test_constant.rbi"
+          assert_equal(expected, result)
+        end
+      end
+
+      describe "#constantize" do
+        it "returns an empty array when constants aren't found" do
+          skip
+          constants = ["Foo", "Bar", "Baz"]
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
+          result = generator.constantize(constants)
+          expected = []
+          assert_equal(expected, result)
+        end
+
+        it "returns an array of filepaths when constants are found" do
+          skip
```

Comment:
Couple of skips here, can you add a comment for why they're skipped?

Reply:
So previous behaviour was to just exit the application when constants weren't found or were un-processable. I want to leave that behaviour to the CLI itself rather than a generator. So wanted to get some feedback before moving forward with a fleshed out solution. In this case, I've simply removed the `exit(1)` call from the original method, but haven't made any other changes.

My proposed solution would be to just return an empty array (as described in the test) for un-processable constants, and potentially log it using the `error_handler` implemented by the subclass.